src/components/ItemTable/ItemAvailability.tsx

```diff
@@ -0,0 +1,93 @@
+import { Text, Link, Button, Box } from "@nypl/design-system-react-components"
+
+import { appConfig } from "../../config/config"
+import type Item from "../../models/Item"
+
+interface ItemAvailabilityProps {
+  item: Item
+}
+
+/**
+ * The ItemAvailability component appears below the Item table and displays
+ * info about an item's availability.
+ * TODO: Add Feedback box, Due date, Available font styles
+ */
+const ItemAvailability = ({ item }: ItemAvailabilityProps) => {
+  // TODO: Move this logic into a getter function in the Item class that returns an availability status key
```

Comment:
Is this follow on work ticketed?

Reply:
This was always a really dicey area. A status key and switch sounds like a great solution.

Reply:
Just ticketed the follow-up work. Not a blocker for this PR.

https://jira.nypl.org/browse/SCC-3906